azuredevops/utils/tfhelper/tfhelper.go

```
@@ -95,3 +95,14 @@ func ParseImportedID(id string) (string, int, error) {
 	}
 	return project, resourceID, nil
 }
+
+// ParseImportedUUID parse the imported uuid from the terraform import
+func ParseImportedUUID(id string) (string, string, error) {
+	parts := strings.SplitN(id, "/", 2)
+	if len(parts) != 2 || parts[0] == "" || parts[1] == "" {
+		return "", "", fmt.Errorf("unexpected format of ID (%s), expected projectid/resourceId", id)
+	}
+	project := parts[0]
+	uuid := parts[1]
```

Perhaps the function should return a parsed UUID and if the string from parts[1] isn't a valid UUID should return an error respectively.

---

All set, it now returns byte[] rather than string.